prometheus-to-sd/translator/prometheus.go

```diff
@@ -36,6 +36,20 @@ type PrometheusResponse struct {
 	rawResponse string
 }
 
+var prometheusClient *http.Client
+
+func init() {
+	// Copy a DefaultTransport that is used by default http client.
+	transport := http.DefaultTransport.(*http.Transport).Clone()
+	// Allow insecure https connections.
+	transport.TLSClientConfig = &tls.Config{
```

Comment:
Should we introduce some timeouts here?
https://medium.com/@nate510/don-t-use-go-s-default-http-client-4804cb19f779

Reply:
There are some timeouts already present in DefaultTransport (that's why I'm using instead of an empty Transport). 
Also have added timeout to the client. I think 30 seconds should be safe to set.